calour/io.py

```diff
@@ -36,13 +35,7 @@ def _read_biom(fp, transpose=True, sparse=True):
     sid = table.ids(axis='sample')
     oid = table.ids(axis='observation')
     logger.info('loaded %d samples, %d observations' % (len(sid), len(oid)))
-    if sparse:
-        logger.debug('storing as sparse matrix')
-        data = scipy.sparse.csr_matrix(table.matrix_data)
-    else:
-        logger.debug('storing as dense matrix')
-        data = table.matrix_data.toarray()
-
+    data = table.matrix_data
```

Comment:
what happened to sparse?
i think it is useful to load experiment sometimes as dense...?